### PR TITLE
[4.0] Dont hide stats data when not needed

### LINKED ISSUE #22980: Select here to see the information that will be sent.
### Steps to reproduce the issue

Edit Joomla stats plugin

### Expected result

Sensible English

A "click" is not a "select", you select an option from a dropdown, you click a link

There is no reason to hide the data to start with? Just show it and remove the link?

### Actual result

> 
Select here to see the information that will be sent. 

screenshot 2018-11-09 at 15 32 35

### Additional comments

related to https://github.com/joomla/joomla-cms/issues/20557 but not a duplicate

### PR DESCRIPTION
**Pull request by PhilETaylor**
Closes #22980 - 

### Summary of Changes

Dont hide data when we dont need to, also negates the need to choose [between "click" and "select"](https://github.com/joomla/joomla-cms/issues/22980#issuecomment-437411686)

### Expected result

screenshot 2018-11-11 at 01 50 08

### Actual result

screenshot 2018-11-09 at 15 32 35

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_system_stats.ini [hunk 1]
```diff
@@ -22,6 +22,7 @@ PLG_SYSTEM_STATS_MODE_OPTION_ON_DEMAND="On demand"
 PLG_SYSTEM_STATS_MSG_ALLOW_SENDING_DATA="Enable Joomla Statistics?"
 PLG_SYSTEM_STATS_MSG_JOOMLA_WANTS_TO_SEND_DATA="To better understand our install base and end user environments it is helpful if you send some site information back to a Joomla! controlled central server. No identifying data is captured at any point. You can change these settings later from Plugins > System - Joomla! Statistics."
 PLG_SYSTEM_STATS_MSG_WHAT_DATA_WILL_BE_SENT="Select here to see the information that will be sent."
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.plg_system_stats.ini:24]: Delete obsolete string?

Comment (PhilETaylor) [reply administrator/language/en-GB/en-GB.plg_system_stats.ini:24]: Not currently obsolete, used elsewhere :)

**Comment by Quy** [comment]
Update `\media\plg_system_stats\js\stats.js` to remove the toggler code?

**Comment by PhilETaylor** [comment]
again, no, Not currently obsolete, used elsewhere :)

There is one other instance of this kind of thing :-)

**Comment by brianteeman** [comment]
not a fan of displaying this by default. what is the benefit

**Comment by brianteeman** [comment]
There is plenty of.reasons to hide it. It is overly technical for the ordinary user. They really only need.to set the plugin options. Anything else is scary and should be optionally displayed only to those that need to see it.

As for click. @mbabker already stated the reason and this change was made a few years ago. I guess you didn't notice.

**Comment by PhilETaylor** [comment]
The " ordinary user" should not be asked a decision like this and should not be editing a plugin they know nothing about, and if the version numbers of your stack are "too technical" then you should not be running Joomla.

Just because a wrong decision has been made doesn't mean it cannot be changed if that decision was wrong. Do your research, you will see article after article on best practice text for links - not one of them will say to do what Joomla currently does (a "select here" on an "a" tag).

# Select here is just wrong. 
You dont select a link, you press, tab. click it. You select from a list or select to highlight text.

# Click here is just as wrong. 

Even w3.org: https://www.w3.org/QA/Tips/noClickHere

[4 Reasons to Avoid Using ‘Click Here’ in Link Text](https://www.wordpress-web-designer-raleigh.com/2015/04/16/4-reasons-to-avoid-using-click-here-in-link-text/)
[links-should-never-say-click-here](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/)
[Why Your Links Should Never Say Click Here](https://www.hypercube.co.nz/links-never-click-here/)
[Why Your Links Should Never Say "Click Here"](http://uxmovement.com/content/why-your-links-should-never-say-click-here/)
[4 Simple Words to Avoid Using on Your Website for Better UX](https://www.protofuse.com/blog/details/words-avoid-using-on-website-better-ux/)
[Why “click here” is a terrible link, and what to write instead](http://stephanieleary.com/2015/05/why-click-here-is-a-terrible-link-and-what-to-write-instead/)

and that was just half of the first page of google....

**Comment by mbabker** [comment]
> if the version numbers of your stack are "too technical" then you should not be running Joomla.

Contrary to your opinion, software like Joomla and WordPress are not exclusive applications that can only be utilized by computer science majors or other trained and/or certified individuals.  You should not bombard someone with technical data they may not understand and tell them "understand it or do not use the software", that's bad UX unless your software is ONLY targeting developers (and even then it's bad unless your target is ONLY developers who are already in the know about whatever the subject is).

**Comment by PhilETaylor** [comment]
The fact is there is no reason to hide this data. It is present elsewhere in Joomla unhidden.

The fact is "select here to view...." is simply, basically, very wrong. Unhiding the data was simple a proposed way to remove the blatantly wrong text. 

Another PR for the bin.

**Pull request closed without merging**